### DISCUSSION TITLE
Dashboard folder is created when an alert folder is created

### DISCUSSION THREAD

**Discussion by OscarVanL** [grafana/grafana#53982] [Platform]
**What happened**:
When creating a folder to store a new alert rule, an API call to POST `/api/folders` is done with that new folder's title. This creates an empty dashboard folder.

**What you expected to happen**:
An alerts folder is created, intuitively there is no need to create a dashboard folder as these are now uncoupled from alerting.

**How to reproduce it (as minimally and precisely as possible)**:
Create an alert folder with a new name, navigate to the dashboards page, an empty dashboard folder now exists.

**Anything else we need to know?**:

**Environment**:
- Grafana version: v9.0.4
- Data source type & version:
- OS Grafana is installed on:
- User OS & Browser:
- Grafana plugins:
- Others:

**Comment by mellieA** [comment]
@OscarVanL I was able to reproduce this behavior following your steps. What I noticed is that the dashboard folder isn't empty, you can view the alerts in the folder under the alerts tab. 

![Screen Shot 2022-08-19 at 1 01 07 PM](https://user-images.githubusercontent.com/18175287/185693158-3209830f-1674-4219-8e4c-721a4e553a5e.png)

Folders hold dashboards, alerts, and library panels. What do you propose that would make this more intuitive? I've moved this conversation to discussions (feature requests) instead of issues (bugs).

**Reply by OscarVanL** [reply]
Hi mellie,

Thanks for the explanation, I can see why this might have made sense with the old grafana alerts approach where alerts were tied to specific dashboards. 

My intuition was that now dashboards and alerts are entirely uncoupled. By that I mean that creating a folder named "A" in the dashboards page of Grafana wouldn't necessarily create a folder "A" on the alerts page, or deleting a folder in the dashboards panel wouldn't delete alerts in the alerts panel. This is because there is no longer a requirement for an alert to relate to any dashboard. 

Now that you've cleared this up for me, I think it is probably fine as it is. It is probably not worth the developer time to change.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
